charts/llm-engine/values_sample.yaml

```diff
@@ -94,15 +135,15 @@ config:
                 "AWS": "arn:aws:iam::985572151633:root"
               },
               "Action": "sqs:*",
-              "Resource": "arn:aws:sqs:us-west-2:985572151633:${queue_name}"
+              "Resource": "arn:aws:sqs:us-east-1:985572151633:${queue_name}"
```

Comment:
What's this account id?

Reply:
This was copy pasted from the ML account,  replaced it with all 0's